dotCMS/src/main/java/com/dotcms/security/apps/AppSchema.java

```diff
@@ -111,9 +113,16 @@ public Map<String, ParamDescriptor> getParams() {
         return new LinkedHashMap<>(params);
     }
 
-    public void addParam(final String name, final Object value, final boolean hidden,
-            final Type type, final String label, final String hint, final boolean required) {
-        params.put(name, ParamDescriptor.newParam(value, hidden, type, label, hint, required));
+    public void addParam(final String name,
+                         final Object value,
+                         final boolean hidden,
+                         final Type type,
+                         final String envVar,
+                         final boolean envShow,
+                         final String label,
+                         final String hint,
+                         final boolean required) {
+        params.put(name, ParamDescriptor.newParam(value, hidden, type, envVar, envShow, label, hint, required));
```

Comment:
Looks like this grew too much. Consider a refactor to a DTO

Reply:
Refactored.